src/Orleans/Runtime/RequestContext.cs

```diff
@@ -111,7 +115,14 @@ public static void Import(Dictionary<string, object> contextData)
                 {
                     activityIdObj = Guid.Empty;
                 }
-#if !NETSTANDARD_TODO
+                else
+                {
+#if NETSTANDARD
```

Comment:
Why add an else here? it alters original logic. The local value in ActivityId would remain unchanged if first `if ` condition succeed.